client/babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    presets: [
+      '@babel/preset-env',
+      '@babel/preset-react',
+    ],
```

Comment:
I see a few of those pesky "no newline at end of file". I'll do a search to see how or if we can get that configured in eslint since right now it's a non blocking thing that pops up in PRs.

I see it in .env, this configuration, stylesMock.js, and jest.config.js.

Reply:
I have been looking into it and the linter should be picking it up, but I am unsure why it is not

Reply:
I added a newline at the end of all the files, but for some reason I still get the no new line at EOF for babel.config.js and jest.config.js

Reply:
^ fixed this